### DISCUSSION TITLE
Custom Popup

### DISCUSSION THREAD

**Discussion by anujdivesh** [TerriaJS/terriajs#5920] [Q&A]
Hi: 
I was wondering if it would be possible to have a custom HTML/JS popup when we click a point in the terriaMap in an iFrame. Is there a way to get these values in the JavaScript?

Thanks in advance.

**Comment by mattpgreen** [comment]
adding to this... has anyone done richer html/css etc coding to ideally enable an iframe containing  another app within the pop up  when  clickinng on a point feature in terriajs ... actual use case is to have pin in TerriaMap that when clicked can open a popup  within  TerriaMap that shows the applicable pano in https://digitaldarwin.au

**Comment by ljowen** [comment]
Hi @mattpgreen any html used for the feature info panel is sanitised which includes removing any `<iframe>` for security reasons. You could have an external link to open the pano in a new tab.
